src/screens/AdminPage.js

```diff
@@ -0,0 +1,20 @@
+// Components
+import { Layout, Table } from 'components';
+
+function App({ darkMode }) {
+  return (
+    <Layout enableFab>
+      <div class='container'>
+        <header style={{ textAlign: 'center' }}>
+          <h3 style={{ margin: '2%' }}>Admin Panel</h3>
+        </header>
+        <Table
+          darkMode={darkMode}
+          style={{ height: '100vh', display: 'flex', flexDirection: 'column' }}
+        />
+      </div>
+    </Layout>
+  );
+}
+
+export default App;
```

Comment:
The table of Admin panel and normal table are different. Admin panel one has edit, extra columns and everything